requirements.txt

```diff
@@ -0,0 +1 @@
+-r requirements/heroku.txt
```

Comment:
My personal preference here would be to move all requirements (including dev and heroku) into a single ``requirements.txt`` (I'm not much of a fan of splitting up the requirements files). I think this would really improve readability and work well with Heroku too.